docker-compose/Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM arm32v6/alpine:latest
 
 COPY --from=builder /tmp/qemu-arm-static /usr/bin
 
-RUN apk add --no-cache py3-pip py3-cffi \
+RUN apk add --no-cache py2-pip py2-cffi \
```

Comment:
I am interested why you downgraded to python 2?

Reply:
Because pip is only available in python 2 in the alpine packages

Reply:
Strange. I wonder why.

Reply:
FYI pip3 gets shipped with python3. Just run ``apk add python3`` and you have it.